public/index.html

```diff
@@ -18,7 +18,7 @@
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
   </head>
-  <body class="announcement-active">
```

Comment:
No. They will be adding announcements in the future. I didn't want the markup hanging out on the page but figured the styles were good to stay.

> On Sep 23, 2015, at 1:57 AM, Chris Van <notifications@github.com> wrote:
> 
> In public/index.html:
> 
> > @@ -18,7 +18,7 @@
> >        <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
> >      <![endif]-->
> >    </head>
> > -  <body class="announcement-active">
> don't you want to remove the .announcement-active styles too?
> 
> —
> Reply to this email directly or view it on GitHub.
>